velox/dwio/parquet/tests/reader/ParquetTableScanTest.cpp

```diff
@@ -116,6 +118,27 @@ class ParquetTableScanTest : public HiveConnectorTestBase {
     assertQuery(plan, splits_, sql);
   }
 
+  void assertSelectWithTimezone(
+      std::vector<std::string>&& outputColumnNames,
+      const std::string& sql,
+      const std::string& sessionTimezone) {
+    auto rowType = getRowType(std::move(outputColumnNames));
+    auto plan = PlanBuilder().tableScan(rowType).planNode();
+    std::vector<exec::Split> splits;
+    splits.reserve(splits_.size());
+    for (const auto& connectorSplit : splits_) {
+      splits.emplace_back(folly::copy(connectorSplit), -1);
+    }
+
+    AssertQueryBuilder(plan, duckDbQueryRunner_)
+        .connectorSessionProperty(
```

Comment:
Interested why we don't have a common function in `assertQuery` which argument hiveConnecterConfigs

Reply:
I can add a `assertQuery` method that supports passing in configs.